gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/extensions/UIUtils.kt

```diff
@@ -0,0 +1,25 @@
+package com.gravatar.quickeditor.ui.extensions
+
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.ui.geometry.Rect
+import androidx.compose.ui.unit.Density
+import androidx.compose.ui.unit.LayoutDirection
+
+internal fun Rect.getPaddedBounds(
```

Comment:
Not a big deal at this point as we only have. this, but I would try to avoid having a file with all possible UI utils. Those can be for very different things. So smaller, specialized utils would be my preference. `RectExtensions/Utils` for example.

Reply:
I'm conflicted about having "too" specialized utils files too 🤔. I'd refactor this later if we have other utils joining the list.